std_msgs/Header header 
geometry_msgs/Pose2D pose
float32[] kernel_param
float32 noise_var
float32 likelihood_var
float32[] inducing_points 

